script/Arcueid/Day 7/07_00_ARC07_3_1_1.txt

```diff
@@ -86,7 +86,7 @@ One leader commanding a large number of worker ants... something along those lin
 -- Page 526, Offset 17877.
 -- 前にも話したけど、こういう階梯」
 // IT'S WHITEBOARD TIME FUCK MY LIFE!!!! ( ͡° ͜ʖ ͡°)
-We touched on this before, but here are the ranks."
+We touched on this before, but here are the Ranks."
```

Comment:
lowercase is fine

Reply:
I think upper case is better for the vampire ranks in particular.